web/src/main/java/org/springframework/security/web/FilterChainProxy.java

```diff
@@ -16,27 +16,33 @@
 
 package org.springframework.security.web;
 
+import java.io.IOException;
```

Comment:
Please remove any unnecessary changes (i.e. formatting) that are not related to the issue.

Reply:
Yeah IDE auto-format, is there a profile I can import? Otherwise it is a PITA to try to get IntelliJ to leave them alone.